src/main/resources/templates/haskell/setup_exercise.sh

```diff
@@ -3,6 +3,9 @@
 # $1: exercise short name as specified on Artemis
 # $2: (optional) output folder name
 
+# base URL to repositories
+BASE="ssh://git@bitbucket.ase.in.tum.de:7999/$EXERCISE/$EXERCISE"
```

Comment:
I would suggest we move this script to the documentation.
@kappelmann would that be ok?

The dependency to one specific version control service might otherwise be confusing here

Reply:
Done.